i18n/join_process.i18n.dart

```diff
@@ -4,16 +4,16 @@ extension Localization on String {
 
 static var _t = Translations.byLocale("en_us") +
     {
-      "es_es": {
-         "Initialize new wallet...": "Inicializar billetera nueva...",
-          "Process invite link...": "Procesar enlace de invitación...",
-          "Accept invite from %s...": "Aceptar invitación de %s...",
+      "pt_br": {
+         "Initialize new wallet...": "Inicializando nova carteira...",
+          "Process invite link...": "Procesar link de convite...",
```

Comment:
Should be "Processar" miss the letter 's'? @juliolrmonteiro

Reply:
Fixed on latest commit, thanks!